### PR TITLE
Connection Tracking Flow Dump Test Case

### PR DESCRIPTION
**Pull request by hirentha**

### CODE REVIEW

**Comment by hirentha** [comment]
Logs for Ct_flow_dump test case,[Ct_flow_dump_logs.txt](https://github.com/ipdk-io/ptf-tests/files/10739753/Ct_flow_dump_logs.txt)

File: tests/DPDK_Flow_Dump_with_CT.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+            ):
+                self.result.addFailure(self, sys.exc_info())
+                self.fail(f"Failed to add table entry {match_action}")
+
+        # Get dump entries table
+        log.info("Get flow dump table entry")
+        dump_table = p4rt_ctl.p4rt_ctl_dump_entities(self.config_data["switch"])
+        # Verify each entry
+        log.info("Verify each table entry")
+        if len(dump_table[1:-1]) != len(self.config_data["flow_dump_table"]):
```
Comment (dkaranjai) [inline tests/DPDK_Flow_Dump_with_CT.py:114]: please explain the slice [1:-1] ... good for reading

**Review by dkaranjai — CHANGES_REQUESTED** [review]

File: tests/DPDK_Flow_Dump_with_CT.py (hunk 1 above)
Comment (hirentha) [reply tests/DPDK_Flow_Dump_with_CT.py:114]: It's already mentioned as comment above that line. It's for Verify each rule.

**Review by sjwahub — APPROVED** [review]

**Review by dkaranjai — APPROVED** [review]
LGTM

**Pull request merged**